dox/dox_only.sh.in

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 # Run doxygen-style documentation system.
-rm -f dddoc_cache.bin && python2 ../util/bin/dox.py -b .. -i ../include/seqan -i pages --image-dir ./images/docs2 --image-dir ./images/reduced_aminoacid/ $*
+rm -f dddoc_cache.bin && python3 ../util/bin/dox.py -b .. -i ../include/seqan -i pages --image-dir ./images/docs2 --image-dir ./images/reduced_aminoacid/ $*
```

Comment:
it is not guaranteed that python (version 3) has an executable python3

Reply:
:+1: